### PR TITLE
refactor(server): graphql schema

### PR DESCRIPTION
**Pull request by yk-eukarya**

### CODE REVIEW

File: server/gqlgen.yml [hunk 1]
```diff
@@ -4,7 +4,19 @@
...
+  - ./gql/team.graphql
+  - ./gql/project.graphql
+  - ./gql/scene.graphql
+  - ./gql/plugin.graphql
+  - ./gql/property.graphql
+  - ./gql/tag.graphql
+  - ./gql/cluster.graphql
+  - ./gql/was.graphql
+  - ./gql/layer.graphql
+  - ./gql/dataset.graphql
```
Comment (rot1024) [inline server/gqlgen.yml:19]:
```suggestion
  - ./gql/*.graphql
```

https://github.com/99designs/gqlgen/issues/467

**Review by rot1024 — CHANGES_REQUESTED** [review]

File: server/gql/_shared.graphql [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+directive @goModel(
+  model: String
+  models: [String!]
+) on OBJECT | INPUT_OBJECT | SCALAR | ENUM | INTERFACE | UNION
+
+directive @goField(
+  forceResolver: Boolean
+  name: String
+) on INPUT_FIELD_DEFINITION | FIELD_DEFINITION
```
Comment (rot1024) [inline server/gql/_shared.graphql:18]:
```suggestion
```

Please deprecate these directive and use `gqlgen.yml` instead to enforce generating resolvers

File: server/gqlgen.yml (hunk 1 above)
Comment (yk-eukarya) [reply server/gqlgen.yml:19]: Done

File: server/gql/_shared.graphql (hunk 2 above)
Comment (yk-eukarya) [reply server/gql/_shared.graphql]: Done

File: server/gql/_shared.graphql [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+  underline: Boolean
+}
+
+enum TextAlign {
+  LEFT
+  CENTER
+  RIGHT
+  JUSTIFY
+  JUSTIFY_ALL
+}
```
Comment (yk-eukarya) [inline server/gql/_shared.graphql:94]: @rot1024 
It seems those are not used types, should I remove them?

Comment (rot1024) [reply server/gql/_shared.graphql:94]: No, they are used

**Review by rot1024 — APPROVED** [review]

**Pull request merged**